subsys/bluetooth/host/hci_core.c

```diff
@@ -1159,6 +1159,7 @@ static void conn_auto_initiate(struct bt_conn *conn)
 
 static void le_conn_complete_cancel(uint8_t err)
 {
+	int retval;
```

Comment:
Suggest to rename to `ret` for consistency with rest of Zephyr codebase.

Reply:
> Suggest to rename to `ret` for consistency with rest of Zephyr codebase.

I prefer `err` myself since it's more intuitive to read `if (err)` than `if (ret)`. The `uint8_t err` parameter should probably be renamed to `uint8_t hci_err`.

Reply:
I now use `ret` here as `err` is an input parameter to the function.